api/pkg/s3/api-response.go

```diff
@@ -0,0 +1,94 @@
+/*
+ * Minio Cloud Storage, (C) 2015, 2016 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package s3
+
+import (
+	"encoding/xml"
+	"net/http"
+
+	"github.com/emicklei/go-restful"
+	. "github.com/journeymidnight/yig/error"
+	"github.com/journeymidnight/yig/helper"
+)
+
+const (
+	timeFormatAMZ = "2006-01-02T15:04:05.000Z" // Reply date format
+)
+
+// writeErrorResponse write error headers
+// w http.ResponseWriter, r *http.Request
+func WriteErrorResponse(request *restful.Request, response *restful.Response, err error) {
+	WriteErrorResponseHeaders(response, err)
+	WriteErrorResponseNoHeader(response, request, err, request.Request.URL.Path)
+}
+
+func WriteErrorResponseHeaders(response *restful.Response, err error) {
+	var status int
+	apiErrorCode, ok := err.(ApiError)
+	if ok {
+		status = apiErrorCode.HttpStatusCode()
+	} else {
+		status = http.StatusInternalServerError
+	}
+	helper.Logger.Println(20, "Response status code:", status, "err:", err)
```

Comment:
We have different logger framework, we can use that

Reply:
Sorry, I will change it with our logger

Reply:
@nguptaopensds thanks for your suggestion, the old logger is used in many files which are ported from yig, so can we make a separate pr to fix this kind of problem instead of this pr?

Reply:
@nguptaopensds the old logger has been removed from this file and you can review it again, thanks.